spigot/src/main/java/com/sekwah/advancedportals/spigot/tags/ConditionsTag.java

```diff
@@ -0,0 +1,120 @@
+package com.sekwah.advancedportals.spigot.tags;
+
+import com.sekwah.advancedportals.core.connector.containers.PlayerContainer;
+import com.sekwah.advancedportals.core.registry.TagTarget;
+import com.sekwah.advancedportals.core.util.Lang;
+import com.sekwah.advancedportals.core.warphandler.ActivationData;
+import com.sekwah.advancedportals.core.warphandler.Tag;
+import com.sekwah.advancedportals.spigot.SpigotInfoLogger;
+import me.clip.placeholderapi.PlaceholderAPI;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+public class ConditionsTag implements Tag.Activation, Tag.Split {
+
+    @Override
+    public boolean preActivated(TagTarget target, PlayerContainer player, ActivationData activeData, String[] argData) {
+
+        for (String condition : argData) {
+            if (!checkConditions(condition, Bukkit.getPlayer(player.getUUID()))) {
+                player.sendMessage(Lang.getNegativePrefix() + Lang.translate("tag.conditions.fail"));
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    @Override
+    public void postActivated(TagTarget target, PlayerContainer player, ActivationData activationData, String[] argData) {
+
+    }
+
+    @Override
+    public boolean activated(TagTarget target, PlayerContainer player, ActivationData activationData, String[] argData) {
+        return false;
+    }
+
+    @Override
+    public TagType[] getTagTypes() {
+        return new TagType[] { TagType.PORTAL };
+    }
+
+    @Override
+    public String getName() {
+        return "conditions";
+    }
+
+    @Override
+    public String[] getAliases() {
+        return new String[0];
+    }
+
+    @Override
+    public String description() {
+        return Lang.translate("tag.conditions.description");
+    }
+
+    private boolean checkConditions(String condition, Player player) {
+        // Remove whitespaces before splitting the condition
+        String trimmedCondition = condition.replaceAll("\\s+", "");
+
+        // Split the condition into placeholder and value parts
+        String[] parts = trimmedCondition.split("<=|>=|<|>|==");
+
+        if (parts.length == 2) {
+            // Trim to remove any leading/trailing whitespaces
+            String placeholder = parts[0].trim();
+            String actualValue = PlaceholderAPI.setPlaceholders(player, placeholder);
+            String restOfCondition = parts[1].trim();
+
+            // Preserve the operator
+            String operator = condition.substring(placeholder.length(), condition.length() - restOfCondition.length()).trim();
+
+            return performComparison(actualValue, operator, restOfCondition);
+        } else {
+            // Log a warning or handle the case where the condition format is invalid

```

Comment:
Check the command portals, could you add a check to make sure that these conditions are met when the tag is being created? then it should theoretically never be able to get here.